BetterMD/elements/table.py

```
@@ -136,12 +189,43 @@ class Table(Symbol):
+
+        super().__init__(styles, classes, inner, **props)
+
+    def prepare(self, parent:'Symbol'=None, dom:'list[Symbol]' = None, *args, **kwargs):
+        if dom is None:
+            dom = []
+        dom.append(self)
+
+        self.prepared = True
+        self.parent = parent
+        for symbol in self.children:
+            symbol.prepare(self, dom.copy(), table=self, *args, **kwargs)
```

Ruff (0.8.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)